scripts/ci/pr_governance_gate.sh

```
@@ -385,8 +412,15 @@ if [ "$CODERABBIT_COUNT" = "0" ]; then
     )"
     if [ "$OPENCODE_ADVERSARIAL_APPROVAL_COUNT" = "0" ]; then
       add_waiting "Waiting for current-head CodeRabbit evidence or a structured OpenCode App adversarial approval on ${HEAD_REF_OID}."
+    elif [ "$CODERABBIT_APPROVAL_PENDING_COUNT" != "0" ]; then
+      # CodeRabbit has no check-run yet but has posted its own approval-pending
+      # issue comment for this exact head: it is actively engaged, not absent,
+      # so a different model's approval must not substitute for its own
+      # terminal verdict. Wait for CodeRabbit specifically.
+      add_waiting "Waiting for CodeRabbit to review the latest commit on ${HEAD_REF_OID}."
     else
-      printf 'CodeRabbit check is absent; accepted current-head OpenCode App adversarial approval on %s.\n' "$HEAD_REF_OID"
+      OPENCODE_FALLBACK_APPROVED=1
```

Shellcheck (0.11.0)

[warning] OPENCODE_FALLBACK_APPROVED appears unused. Verify use (or export if used externally).

(SC2034)